common/test/views/support/cleaner/CommercialMPUForFrontsTest.scala

```diff
@@ -0,0 +1,167 @@
+          </section>
+          <section class="fc-container">
+            <div class="fc-container__inner">
+              <div class="fc-slice-wrapper">
+                <ul class="u-unstyled l-row">
+                  <li class="fc-slice__item l-row__item"></li>
+                  <li class="fc-slice__item l-row__item"></li>
+                  <li class="fc-slice__item l-row__item"></li>
+                  <li class="fc-slice__item l-row__item fc-slice__item--mpu-candidate"></li>
+                </ul>
+              </div>
+            </div>
+          </section>
+          <div class="fc-container fc-container--commercial">
+            <div id="dfp-ad--merchandising-high" class="ad-slot--merchandising-high">
+            </div>
+          </div>
+          <section class="fc-container">
+            <div class="fc-container__inner">
+              <div class="fc-slice-wrapper">
+                <ul class="u-unstyled l-row">
+                  <li class="fc-slice__item l-row__item"></li>
+                  <li class="fc-slice__item l-row__item"></li>
+                  <li class="fc-slice__item l-row__item"></li>
+                  <li class="fc-slice__item l-row__item"></li>
+                </ul>
+              </div>
+            </div>
+          </section>
+          <section class="fc-container">
+            <div class="fc-container__inner">
+              <div class="fc-slice-wrapper">
+                <ul class="u-unstyled l-row">
+                  <li class="fc-slice__item l-row__item"></li>
+                  <li class="fc-slice__item l-row__item"></li>
+                  <li class="fc-slice__item l-row__item"></li>
+                  <li class="fc-slice__item l-row__item"></li>
+                </ul>
+              </div>
+            </div>
+          </section>
+          <section class="fc-container">
+            <div class="fc-container__inner">
+              <div class="fc-slice-wrapper">
+                <ul class="u-unstyled l-row">
+                  <li class="fc-slice__item l-row__item"></li>
+                  <li class="fc-slice__item l-row__item"></li>
+                  <li class="fc-slice__item l-row__item"></li>
+                  <li class="fc-slice__item l-row__item fc-slice__item--mpu-candidate"></li>
+                </ul>
+              </div>
+            </div>
+          </section>
+          <section class="fc-container">
+            <div class="fc-container__inner">
+              <div class="fc-slice fc-slice--popular">
+              </div>
+            </div>
+          </section>
+          <div class="fc-container fc-container--commercial">
+            <div id="dfp-ad--merchandising" class="ad-slot--merchandising ad-slot--commercial-component">
+            </div>
+          </div>
+        </body>
+      </html>.toString()
+
+  it should "insert MPUs into applicable slices, and give them unique IDs" in {
+    val result = parseTestData(frontWithThrasher)
+    val body = clean(result)
```

Comment:
you can extract those 2 lines outside of the tests itself and keep it at the class level so it is executed only once